rustls/src/server/tls13.rs

```diff
@@ -696,9 +696,12 @@ mod client_hello {
         cr.extensions
             .push(CertReqExtension::SignatureAlgorithms(schemes.to_vec()));
 
-        let names = config
+        let names: Vec<PayloadU16> = config
             .verifier
-            .client_auth_root_subjects();
+            .client_auth_root_subjects()
+            .iter()
+            .map(|n| PayloadU16::new(n.clone()))
+            .collect();
```

Comment:
I'd typically use `collect::<Vec<_>>()` which avoids naming the type of the element twice (and I have a somewhat irrational dislike of type annotations).